### ISSUE TITLE
Other tedx pages

### ISSUE THREAD

**Issue by khannajai** [TEDxJacobsUniversity/static_page#1]
Hey everyone,
I've created this issue for you all to give some input about what you like or dislike about certain tedx websites.

In your posts, do try and mention the following points:
What you like about a specific website. Why?
What you dislike about a specific website. Why?

Looking forward to your responses.

**Comment by khannajai** [comment]
Went through at a couple of different websites and this is one that I particularly liked
[tedx Portland](http://www.tedxportland.com/):
A simple, uncluttered, and easy to use website. In my opinion, quite well designed. Everything is laid out on one page and you can simply navigate to whichever year's tedx you want using the tabs on top. The focus is on the speakers, which is what I think most people are interested in. You can simply click on any speaker and immediately access their tedx talk. 
What the website lacks is sufficient and easily accessible information about the speakers. It would be cool if they had drop-down sections with information about each speaker/talk upon clicking on each speaker. Also, maybe, an about, sponsors and contact section at the bottom would be helpful.

**Comment by thunderboltsid** [comment]
@khannajai That's a very nice and elegant design. Straight, simple, and delivers the whole point of the conference by wrapping all the talks of past years in one page. Interestingly, at TEDxJUB, there's this problem that the pages have been differently designed every year because the people making them were different and everybody decided to trash the old page and make a new one. I can see two solutions to make it work. 
1. Have a landing page that just links to different editions of TEDxJUB ala https://tedxjacobsuniversity.github.io/static_page/ and then just link to the old sites and the new site
2. Do a major overhaul and try to condense previous editions into one single page site ala TEDxPDX

What do you guys think @musikers @sigma271 @ReoCSun ?

**Comment by ruicao-github** [comment]
I think to make a linking page is better which requires less work. Also by do so, we perserve the work of previous generations, which is kind of showing respect to them. It will be great that after 50 years, we can look back to prevous works from every generation.

**Comment by thunderboltsid** [comment]
@musikers Yes, with doing so we will preserve the old work but we should be asking ourselves `do we really care?`. At the moment it's like this. The current year's initial template is here https://tedxjacobsuniversity.github.io/static_page/2016/ 
I'm quite open to having changes. Let me know what you guys think of it.
